src/openloco/graphics/gfx.cpp

```diff
@@ -184,10 +184,82 @@ namespace openloco::gfx
      */
     uint16_t get_string_width(const char* buffer)
     {
-        registers regs;
-        regs.esi = (uintptr_t)buffer;
-        call(0x495685, regs);
-        return regs.cx;
+        uint16_t width = 0;
+        uint8_t* str = (uint8_t*)buffer;
+
+        while (*str != (uint8_t)0)
+        {
+            uint8_t chr = *str;
+            str++;
+
+            if (chr >= 32)
+            {
+                width += _characterWidths[chr - 32 + _currentFontSpriteBase];
```

Comment:
use a local copy of spritebase.

Reply:
I did it. Should be a default initial value or we just copy it from that variable?